content/articles/2018-02-tidyr-0-8-0.Rmd

```diff
@@ -0,0 +1,51 @@
+---
+title: tidyr 0.8.0
+author: Hadley Wickham
+date: '2018-02-13'
+slug: tidyr-0-8-0
+categories:
+  - package
+tags: []
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+library(tidyr)
+library(tibble)
+```
+
+We are pleased to announce that tidyr 0.8.0 is now available on CRAN. tidyr makes it easy to “tidy” your data, storing it in a consistent form so that it’s easy to manipulate, visualise and model. Tidy data has a simple convention: put variables in the columns and observations in the rows. You can learn more about it in the [tidy data vignette](http://tidyr.tidyverse.org/articles/tidy-data.html). Install it with:
+
+```{r, eval = FALSE}
+install.packages("tidyr")
+```
+
+This release mainly contains a bumper crop of small bug fixes and minor improvements, and a considerably increase in test coverage (84% to 99%). For the full details, see the [release notes](https://github.com/tidyverse/tidyr/releases/tag/v0.8.0). Here we'll highlight an important bug fix that might change existing code, and one new feature you might like to try out.
+
+## API changes
+
+There was a bug in `separate()` where negative values had an off by one error. Now -1 correctly refers to the first position between characters counting from the right hand side.
```

Comment:
`off-by-one error` (good ol' compound modifier strikes again...even wikipedia agrees with me here: https://en.wikipedia.org/wiki/Off-by-one_error). RHS is always dealers choice (righthand, right-hand, and right hand are all used interchangeably).